### PR TITLE
Add documentation for Loop accelerator

### PR DESCRIPTION
**Pull request by eleftherias**
Which other branches should this be merged with (if any)?
Only 1.4

### CODE REVIEW

**Comment by RichardJJG** [comment]
Hi, @eleftherias , do you have an ETA for when this will be ready to merge? I ask because the longer this stays in limbo, the more difficult it will be to merge.

**Comment by eleftherias** [comment]
Hey @RichardJJG, I estimate it will be ready to merge by the end of the week.

**Comment by RichardJJG** [comment]
Thanks, @eleftherias, I've created a note on the [story](https://www.pivotaltracker.com/story/show/183581479) for a writer to check back on 7 November.

File: application-accelerator/creating-accelerators/transforms/loop.hbs.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+`doAsMerge` executes the transform on the same input files for every iteration and merges the resulting outputs. It is
+best suited when a transform should be executed multiple times on the same input and the outputs will not conflict.
+
+`doAsChain` executes the transform on the initial input files once and then passes the resulting output to the second
+iteration and so on. It is best suited when a transform needs to be aware of any changes that occurred in the previous
+iteration.
+
+## <a id="examples"></a>Examples
+
+Create a new directory for every module in `modules` containing the contents of the "template" directory.
```
Comment (ericbottard) [inline application-accelerator/creating-accelerators/transforms/loop.hbs.md:47]:
```suggestion
Create a new directory for every module in `modules` (a list of strings) based on the contents of the "template" directory.
```

File: application-accelerator/creating-accelerators/transforms/loop.hbs.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+type: Loop
+on: "#modules"
+var: m
+doAsChain:
+  type: RewritePath
+  regex: "template/(.*)"
+  rewriteTo: "#m + '/' + #g1"
+```
+
+Add every artifact in `artifacts` as a Spring plugin.
```
Comment (ericbottard) [inline application-accelerator/creating-accelerators/transforms/loop.hbs.md:59]:
```suggestion
Add every artifactId in `artifacts` (a list of strings) as a Spring plugin.
```

**Review by ericbottard — CHANGES_REQUESTED** [review]
LGTM with the minor changes suggested

**Pull request merged**